lua/wire/zvm/zvm_core.lua

```diff
@@ -269,7 +269,7 @@ function ZVM:Dyn_EmitInterrupt(intNo,intParam)
   self:Dyn_EmitState()
   self:Emit("VM.IP = %d",(self.PrecompileIP or 0))
   self:Emit("VM.XEIP = %d",(self.PrecompileTrueXEIP or 0))
-  self:Dyn_Emit("VM:Interrupt(%d,%d)",intNo,intParam)
+  self:Dyn_Emit("VM:Interrupt(%s,%s)",intNo,intParam)
```

Comment:
In the case of the page functions, it seems to be because IDX is a local variable

SPG defines it as such for example:
self:Dyn_Emit("$L IDX = math.floor($1 / 128)")
where $L is short for local and $1 is the macro for the first operand, these go through the preprocessor only after going through dyn_emit's string.format so I don't exactly know of a way to coerce these to raw numbers at the instruction level, if we could guarantee that $1 and $2 are table values that can be accessed via quoted strings we could replace %s with %q, which might be safer.

Reply:
It'd be ideal if they were all changed to be number. We want to minimize '%s' in string.formats for security. '%q' isn't any more secure.